package.json

```diff
@@ -72,7 +74,7 @@
     "watch": "npm-watch"
   },
   "dependencies": {
-    "@salesforce/refocus-collector": "^1.2.1",
+    "@salesforce/refocus-collector": "^1.3.0",
```

Comment:
??? - the collector is currently at version 1.2.1

Reply:
i dunno i just ran npm update on this

Reply:
I'm not sure why npm did this, but I think you need to back out this change, since there is no module matching that version spec -- https://www.npmjs.com/package/@salesforce/refocus-collector

Reply:
this has already been reverted @iamigo